packages/web-app-files/src/components/BatchActions.vue

```diff
@@ -1,118 +1,150 @@
 <template>
-  <div class="uk-flex uk-flex-middle">
+  <ul
```

Comment:
You could use the `oc-list` component here. Should do the same, but without UiKit.

Reply:
`oc-list` also has zero margin (which will get interesting to sort out during the UiKit removal...)

Reply:
Pretty sure this has been resolved by you by now, marking as resolved